config/deploy.rb

```diff
@@ -44,3 +44,79 @@
     end
   end
 end
+
+namespace :container do
+  desc 'Hello, world!'
+  task :hello do
+    on roles(:container) do
+      execute "echo Hello, world!"
+    end
+  end
+
+  desc 'deploy'
+  task :deploy do
+    on roles(:container) do
+      execute "sudo docker pull ryuzee/slidehub:latest"
+      prefix = DateTime.now.strftime('%Y%m%d%H%M%s')
+      cmd =<<"EOS"
```

Comment:
__[rubocop]__
* Style/SpaceAroundOperators: Surrounding space missing for operator `=`. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)